src/jit/codegenxarch.cpp

```diff
@@ -5329,13 +5331,13 @@ void CodeGen::genCallInstruction(GenTreeCall* call)
     // Is the caller supposed to pop the arguments?
     if (fCallerPop && (stackArgBytes != 0))
     {
-        genAdjustSP(stackArgBytes);
+        stackAdjustBias = stackArgBytes;
```

Comment:
It seems that ``genRemoveAlignmentAfterCall`` is available for all platforms currently (I checked ``jit/codegenlinear.h``).